content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -73,21 +73,19 @@ knows how to convert between them in both directions.  Additionally, any new
 field added in v2 must be able to round-trip to v1 and back, which means v1
 might have to add an equivalent field or represent it as an annotation.
 
-**Rule #3: An API version in a given track may not be deprecated until a new
-API version at least as stable is released.**
+**Rule #3: An API version in a given track may not be deprecated in favor of a less stable API version.**
 
-GA API versions can replace GA API versions as well as beta and alpha API
-versions.  Beta API versions *may not* replace GA API versions.
+  * GA API versions can replace beta and alpha API versions.
+  * Beta API versions can replace earlier beta and alpha API versions, but *may not* replace GA API versions.
+  * Alpha API versions can replace earlier alpha API versions, but *may not* replace GA or beta API versions.
 
-**Rule #4a: Other than the most recent API versions in each track, older API
-versions must be supported after their announced deprecation for a duration of
-no less than:**
+**Rule #4a: minimum API lifetime is determined by the API stability level**
 
-   * **GA: 12 months or 3 releases (whichever is longer)**
-   * **Beta: 9 months or 3 releases (whichever is longer)**
-   * **Alpha: 0 releases**
```

Comment:
nit: line 84, Rule 4a, capitalize "may" or consistently capitalize the rules?

If an API reaches GA, but is then deprecated, can the API ever be removed?

Reply:
> nit: line 84, Rule 4a, capitalize "may" or consistently capitalize the rules?

I'm not seeing the inconsistent capitalization... am I missing it?

> If an API reaches GA, but is then deprecated, can the API ever be removed?

Not without invoking the exceptions clause at the end of this doc. We can mark an API as deprecated to indicate it is not recommended for use and will not receive additional enhancements, but removal of stable APIs would be disallowed by default.

Reply:
nit (very minor): Compare Rule #3 and Rule #4a

Reply:
ah, I see... tweaked format of 4a to match